core/WORKSPACE

```diff
@@ -8,176 +8,4 @@ local_repository(
-    name = "attribute-test",
-    attribute_path = "hello",
-    repository = "@nixpkgs",
-)
-
-nixpkgs_package(
-    name = "expr-attribute-test",
-    attribute_path = "hello",
-    nix_file_content = "import <nixpkgs> { config = {}; overlays = []; }",
-    repository = "@nixpkgs",
-)
-
-nixpkgs_package(
-    name = "nix-file-test",
-    attribute_path = "hello",
-    nix_file = "//tests:nixpkgs.nix",
-    repository = "@nixpkgs",
-)
-
-nixpkgs_package(
-    name = "nix-file-deps-test",
-    nix_file = "//tests:hello.nix",
-    nix_file_deps = ["//tests:pkgname.nix"],
-    repository = "@nixpkgs",
-)
-
-nixpkgs_package(
-    name = "output-filegroup-test",
-    nix_file = "//tests:output.nix",
-    repository = "@nixpkgs",
-)
-
-nixpkgs_package(
-    name = "extra-args-test",
-    nix_file_content = """
-{ packagePath }: (import <nixpkgs> { config = {}; overlays = []; }).${packagePath}
-    """,
-    nixopts = [
-        "--argstr",
-        "packagePath",
-        "hello",
-    ],
-    repository = "@nixpkgs",
-)
-
-nixpkgs_package(
-    name = "output-filegroup-manual-test",
-    build_file_content = """
-package(default_visibility = [ "//visibility:public" ])
-filegroup(
-    name = "manual-filegroup",
-    srcs = glob(["hi-i-exist", "hi-i-exist-too", "bin/*"]),
-)
-""",
-    nix_file = "//tests:output.nix",
-    repository = "@nixpkgs",
-)
-
-nixpkgs_package(
-    name = "relative-imports",
-    attribute_path = "hello",
-    nix_file = "//tests:relative_imports.nix",
-    nix_file_deps = [
-        "//:flake.lock",
-        "//:nixpkgs.nix",
-        "//tests:relative_imports/nixpkgs.nix",
-    ],
-    repository = "@nixpkgs",
-)
-
-# This bazel build @output-derivation-is-a-file//... must fail
-# See https://github.com/tweag/rules_nixpkgs/issues/99 for details
-nixpkgs_package(
-    name = "output-derivation-is-a-file",
-    attribute_path = "",
-    nix_file_content = """let pkgs = import <nixpkgs> { config = {}; overlays = []; }; in pkgs.writeText "foo" "bar" """,
-    nix_file_deps = [
-        "//:flake.lock",
-        "//:nixpkgs.nix",
-        "//tests:relative_imports/nixpkgs.nix",
-    ],
-    repository = "@nixpkgs",
-)
-
-local_repository(
-    name = "nixpkgs_location_expansion_test_file",
-    path = "tests/location_expansion/test_repo",
-)
-
-nixpkgs_package(
-    name = "nixpkgs_location_expansion_test",
-    build_file_content = "exports_files(glob(['out/**']))",
-    nix_file = "//tests:location_expansion.nix",
-    nix_file_deps = [
-        "//tests:location_expansion/test_file",
-        "@nixpkgs_location_expansion_test_file//:test_file",
-    ],
-    nixopts = [
-        "--arg",
-        "local_file",
-        "$(location //tests:location_expansion/test_file)",
-        "--arg",
-        "external_file",
-        "$(location @nixpkgs_location_expansion_test_file//:test_file)",
-    ],
-    repository = "@remote_nixpkgs",
-)
-
-### end dependencies for tests
+rules_nixpkgs_dependencies(toolchains = [])
```

Comment:
I was going to ask why pass `toolchains = []` explicitly instead of relying on the default of `None`, but then I saw `None` means load all toolchains.